website/server/controllers/api-v3/user.js

```diff
@@ -27,16 +29,18 @@ let api = {};
  */
 api.getUser = {
   method: 'GET',
-  middlewares: [authWithHeaders()],
+  middlewares: [authWithHeaders(), locals],
   url: '/user',
   async handler (req, res) {
     let user = res.locals.user;
     let userToJSON = user.toJSON();
+    let env = res.locals.habitrpg;
 
     // Remove apiToken from response TODO make it private at the user level? returned in signup/login
     delete userToJSON.apiToken;
 
     user.addComputedStatsToJSONObj(userToJSON);
+    userToJSON.flags.newStuff = moment(user.flags.newStuffSeenDate).isBefore(env.NEWS_DATE);
```

Comment:
👍 I don't think we want the entire locals middleware just for this, since it includes a lot of things only used for Jade views